azure_monitor/src/azure_monitor/sdk/auto_collection/live_metrics/exporter.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+import collections
+import logging
+import typing
+
+from opentelemetry.sdk.metrics import Counter, Observer
+from opentelemetry.sdk.metrics.export import (
+    MetricRecord,
+    MetricsExporter,
+    MetricsExportResult,
+)
+
+from azure_monitor.protocol import (
+    Envelope,
+    LiveMetric,
+    LiveMetricDocument,
+    LiveMetricDocumentProperty,
+    LiveMetricEnvelope,
+)
+from azure_monitor.sdk.auto_collection import live_metrics
+from azure_monitor.sdk.auto_collection.live_metrics.sender import (
+    LiveMetricsSender,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class LiveMetricsExporter(MetricsExporter):
+    """Live Metrics Exporter
+
+    Export data to Azure Live Metrics service and determine if user is subscribed.
+    """
+
+    def __init__(self, instrumentation_key):
+        self._instrumentation_key = instrumentation_key
+        self._sender = LiveMetricsSender(self._instrumentation_key)
+        self.subscribed = True
+        self._document_envelopes = collections.deque()
+
+    def add_document(self, envelope: Envelope):
```

Comment:
This will be used when actual metrics are sent correct?

Reply:
Yes, ex: exceptions events, failed dependencies, etc.